client/post-editor/post-editor.jsx

```diff
@@ -854,18 +857,11 @@ const PostEditor = React.createClass( {
 	},
 
 	getEditorMode: function() {
-		var editorMode = 'tinymce'
-		if ( this.props.preferences ) {
-			if ( this.props.preferences[ 'editor-mode' ] ) {
-				editorMode = this.props.preferences[ 'editor-mode' ];
-			}
-
-			if ( ! this.recordedDefaultEditorMode ) {
-				analytics.mc.bumpStat( 'calypso_default_editor_mode', editorMode );
-				this.recordedDefaultEditorMode = true;
-			}
+		const editorMode = this.props.editorModePreference || 'tinymce';
+		if ( ! this.recordedDefaultEditorMode ) {
```

Comment:
Thanks! That logic was very well hidden! I rewrite it a bit to behave in the same way and maybe a bit clearer.